general/README.md

```diff
@@ -257,3 +257,17 @@ function getList(
 	parameter3
 ) {}
 ```
+
+
+## Add spaces between curly braces and variable names
```

Comment:
This seems very specific to javascript or even just Angular (and other MVC's), why is this placed in general?